README.md

```diff
@@ -29,13 +66,13 @@ Steps:
 
 Query:
 ```terraform
-avg(${var.containers_evaluation_period}):sum:datadog.estimated_usage.containers{${local.containers_filter}} > ${var.containers_critical}
+avg(last_1h):sum:datadog.estimated_usage.containers{tag:xxx} > 
```

Comment:
It feels a bit weird to leave out any threshold placeholder. Is this on purpose?

Reply:
I'm guessing this is because we don't have a default value there. The docs here are generated. I'll have a look at the script